package.json

```diff
@@ -76,7 +69,7 @@
         "env",
         {
           "targets": {
-            "node": "6.10"
+            "node": "8.0"
```

Comment:
I think this would be a breaking change as anyone running node 6 could see runtime errors depending on what code babel outputs.

We should update the target to 8.0 but probably once we're ready to release a v3.0, I created #59 to track this.

Could you keep the node target at 6.10 in this PR?

Reply:
As you can see in sindresorhus/term-img#9, `term-img` now requires Node 8, so that's why I updated the Node version here too.

Reply:
Ah got it, okay then I'll bump to 3.0 when releasing these changes.